docs/sections/user_guide/yaml/components/chgres_cube.rst

```diff
@@ -0,0 +1,31 @@
+.. _chgres_cube_yaml:
+
+chgres_cube
+===========
+
+Structured YAML to run :ufs-utils:`chgres_cube<chgres-cube>` is validated by JSON Schema and requires the ``chgres_cube:`` block, described below. If ``chgres_cube`` is to be run via a batch system, the ``platform:`` block, described :ref:`here <platform_yaml>`, is also required.
+
+Here is a prototype UW YAML ``chgres_cube:`` block, explained in detail below:
+
+.. highlight:: yaml
+.. literalinclude:: ../../../../shared/chgres_cube.yaml
```

Comment:
The FV3 YAML doc mentions that a `datetime` object called `cycle` is made available to Jinja2 when realizing the FV3 component config, though the information is maybe in a too-specific section, `files_to_copy`. I wonder if it would be helpful to elevate that information, as it actually (IIRC) applies anywhere in the component-specific YAML block (so, anywhere under `chgres_cube:`, `fv3:`, etc. If that got lifted up out of the `files_to_copy` block in the FV3 YAML doc, we could copy/paste it here, which might be helpful given that we're using `cycle` in the `chgres_cube` YAML example.

(BTW, @elcarpenterNOAA and I found a bug in how `cycle` is being processed at the moment. It doesn't affect any of our documented examples, but it came up in practice. I think I know how to fix it and will try to roll it into my upcoming omnibus PR.)